test/models/actions-inputs-model.test.js

```diff
@@ -4,56 +4,58 @@ const allParcels = [{ ref: 'AAA', description: 'Test parcel', totalPerimeter: 10
-    const model = actionsInputsModel('', '', error).model
+    const { model } = actionsInputsModel('', {}, error)
     expect(model.errorMessage).toEqual(
       expect.objectContaining({ text: error })
     )
   })
 
   test('if error isn\'t provided, errorMessage should be omitted', () => {
-    const model = actionsInputsModel('', '').model
+    const { model } = actionsInputsModel('', {})
     expect(model.errorMessage).toBeUndefined()
   })
 
   test('label class set to govuk-label--xl', () => {
-    const model = actionsInputsModel('', '').model
+    const { model } = actionsInputsModel('', {})
     expect(model.label.classes).toBe('govuk-label--xl')
   })
 
   test('label isPageHeading is true', () => {
-    const model = actionsInputsModel('', '').model
+    const { model } = actionsInputsModel('', {})
     expect(model.label.isPageHeading).toBeTruthy()
   })
 
   test('id is actionInput', () => {
-    const model = actionsInputsModel('', '').model
+    const { model } = actionsInputsModel('', {})
     expect(model.id).toBe('actionInput')
   })
 
   test('name is actionInput', () => {
-    const model = actionsInputsModel('', '').model
+    const { model } = actionsInputsModel('', {})
     expect(model.label.classes).toBe('govuk-label--xl')
   })
 
   test('title and hint refers to fencing for Action ID FG1', () => {
-    const model = actionsInputsModel('', 'FG1').model
+    const parcelRef = 'AB1234'
+    const { model } = actionsInputsModel(parcelRef, { id: 'FG1' })
     expect(model.label.text).toBe('Enter a fence length in metres')
-    expect(model.hint.text).toBe('How long is the fence you want to claim on in metres for ?')
+    expect(model.hint.text.trim()).toBe(`How long is the fence you want to claim on in metres for ${parcelRef}?`)
   })
 
-  test('title and hintrefers to acres for Action ID SW6', () => {
-    const model = actionsInputsModel('', 'SW6').model
+  test('title and hint refers to acres for Action ID SW6', () => {
+    const parcelRef = 'AB1234'
+    const { model } = actionsInputsModel(parcelRef, { id: 'SW6' })
     expect(model.label.text).toBe('Enter the area in hectares')
-    expect(model.hint.text).toBe('What is the area you wish to claim on in hectares for ?')
+    expect(model.hint.text.trim()).toBe(`What is the area you wish to claim on in hectares for ${parcelRef}?`)
   })
 
   test('Parcel data is included in the model when a parcel is included', () => {
-    const parcelModel = actionsInputsModel('AAA', 'FG1', '', allParcels).parcelModel
+    const { parcelModel } = actionsInputsModel('AAA', { id: 'FG1' }, '', allParcels)
     expect(parcelModel.caption).toBe('Parcel Information')
   })
 
   test('Parcel data is not included in the model when a parcel is not included', () => {
-    const parcelModel = actionsInputsModel('AAA', 'FG1', '', allParcels).parcelModel
+    const { parcelModel } = actionsInputsModel('AAA', { id: 'FG1' }, '', allParcels)
     expect(parcelModel).toMatchObject({})
```

Comment:
Not added in this PR, but I don't think this test is meaningful. All it shows is that `parcelModel` is an object, which was already proved by the test above using exactly the same call to `actionInputsModel()`